iot-e2e-tests/jvm/src/test/java/tests/integration/com/microsoft/azure/sdk/iot/iothubservices/twin/GetTwinModuleJVMRunner.java

```diff
@@ -34,9 +35,17 @@ public GetTwinModuleJVMRunner(String deviceId, String moduleId, IotHubClientProt
     public static Collection inputs() throws Exception
     {
         iotHubConnectionString = Tools.retrieveEnvironmentVariableValue(TestConstants.IOT_HUB_CONNECTION_STRING_ENV_VAR_NAME);
-        Collection inputs = DeviceTwinCommon.inputsCommon(ClientType.MODULE_CLIENT);
-        identities = getIdentities(inputs);
-        return inputs;
+        isBasicTierHub = Boolean.parseBoolean(Tools.retrieveEnvironmentVariableValue(TestConstants.IS_BASIC_TIER_HUB_ENV_VAR_NAME));
+        if (!Boolean.parseBoolean(Tools.retrieveEnvironmentVariableValue("IS_BASIC_TIER_HUB")))
```

Comment:
>!Boolean.parseBoolean(Tools.retrieveEnvironmentVariableValue("IS_BASIC_TIER_HUB")) [](start = 12, length = 82)

reuse the above env var instead of retrieving it again